src/test/java/org/jitsi/meet/test/misc/SingleModeratorTest.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright @ 2018 8x8 Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.meet.test.misc;
+
+import org.jitsi.meet.test.*;
+import org.jitsi.meet.test.base.*;
+import org.jitsi.meet.test.web.*;
+import org.testng.*;
+import org.testng.annotations.*;
+
+import static org.testng.Assert.*;
+
+/**
+ * This test depends on several external configs, providing valid jwt tokens and a server module that allow a single
+ * moderator in the room.
+ * - single moderator mode is enabled and in the personal room of the participant he/she is the only moderator
+ *   if anyone joins, joins as guest.
+ *
+ * @author Damian Minkov
+ */
+public class SingleModeratorTest
+    extends WebTestBase
+{
+    /**
+     * Required configuration keys.
+     */
+    private static final String TENANT_NAME_PNAME = "org.jitsi.misc.single.tenant_name";
```

Comment:
Wouldn't it make sense to extract a base class for these new tests in order to avoid repetition?